### ISSUE TITLE
(Vue) Custom Parameters are not working

### ISSUE THREAD

**Issue by PhillippOhlandt** [storybooks/storybook#5874]
**Describe the bug**
Hello,

I have a few stories where I don't need the panel because they act as content pages. 

**To Reproduce**
Here is one typescript module:

```typescript
import { storiesOf } from '@storybook/vue';

storiesOf('Specification', module)
  .addParameters({ options: { showPanel: false } })
  .add('Downloads', () => ({
    template: `
      <div style="padding: 20px; color: black; background: white;">
        <h1>Foo</h1>
      </div>
    `,
  }), { options: { showPanel: false } });

```

**Expected behavior**
When I browse to the story, the panel on the bottom should not be visible.

**Current behavior**
The panel stays visible. I also tried other options, e.g. setting the panel position to the right side, which also doesn't work.

**System:**
 - OS: macOS
 - Device: Macbook Pro 2015
 - Browser: Chrome
 - Framework: vue
 - Version: `"@storybook/vue": "4.2.0-alpha.6",`

**Comment by shilman** [comment]
Hi @PhillippOhlandt try this with the newly released `@storybook/vue@5.0.0`. I think you're looking at a new version of the docs but using those options with an old version of Storybook.

**Comment by PhillippOhlandt** [comment]
@shilman That could be it. Just saw that we have an open PR which updates to storybook v5 so I guess it will work as soon as it's merged. Closing this for now.
